packages/terra-tag/src/Tag.scss

```diff
@@ -0,0 +1,71 @@
+
+:local {
+  .tag {
```

Comment:
Do we need to add `font-family: inherit`?

Reply:
shouldn't need to - would be redundant.